.github/workflows/tests-windows.yml

```diff
@@ -21,13 +21,13 @@ jobs:
         - python-version: "3.10"
           env:
             TOXENV: py
-        - python-version: "3.10"
```

Comment:
Not sure why we had asyncio for 3.10 and 3.11, if this was on purpose please chime in.

Reply:
I had a look, and I believe it was by accident, https://github.com/scrapy/scrapy/commit/b6541830849a6edd41da6b539d272790a4661d7d#diff-ab5a8d3ff84c9430ab7fb358981634187b850ef3a5369ee94c332d59fd79dd93 followed by https://github.com/scrapy/scrapy/commit/8a0a9e6d3e5398b884e92cb9336efe3fd19677d6.